### ISSUE TITLE
Does not select '0' as an option (if integer)

### ISSUE THREAD

**Issue by AhsanAyaz** [inmagik/ionic-modal-select#31]
If you provide numbers as options and 0 has to be one of the options, it is never selected because of the check if(!option)

**Comment by bianchimro** [comment]
Hi @AhsanAyaz , thanks for reporting!
It should be a quick fix, I'm going to fix it as soon as possible!

**Comment by bianchimro** [comment]
@AhsanAyaz should be fixed in master. can you confirm that it works for your use case?

**Comment by AhsanAyaz** [comment]
@bianchimro  Sure I'll check and let you know in a few hours. 
What I did at my end was that I changed the options to string values :) So it worked. 
Btw i was working on a project and it had options from -20 to +20 with an interval of 0.25
The requirement was having "0" at middle so I worked on scrolling to a particular element. And it works fine. 
I still need to clean up some code. Will fork your repo and will ping you when I'm done. 
Might be used to navigate directly to the previously selected object by scrolling to the element in future.
Will let you know. 
Thanks for fixing the issue :+1:
